.codegen.json

```diff
@@ -23,7 +23,8 @@
             "make schema",
             "echo 'bundle/internal/tf/schema/\\*.go linguist-generated=true' >> ./.gitattributes",
             "echo 'go.sum linguist-generated=true' >> ./.gitattributes",
-            "echo 'bundle/schema/jsonschema.json linguist-generated=true' >> ./.gitattributes"
+            "echo 'bundle/schema/jsonschema.json linguist-generated=true' >> ./.gitattributes",
+            "pushd experimental/python && uv lock"
```

Comment:
I don't think this is run as part of the tagging workflow, only as part of the generation workflow.

@andrewnester Can you confirm?